lib/evss/common_service.rb

```diff
@@ -8,11 +8,11 @@ class CommonService < BaseService
     BASE_URL = "#{Settings.evss.url}/wss-common-services-web-#{API_VERSION}/rest/"
 
     def create_user_account
-      post 'persistentPropertiesService/11.0/createUserAccount'
+      post 'persistentPropertiesService/11.6/createUserAccount'
```

Comment:
Should we just use the `API_VERSION` constant in this file instead of hard coding the version in-string?